src/RestApi/StoreApi/Schemas/OrderItemSchema.php

```diff
@@ -0,0 +1,280 @@
+						],
+						'alt'  => [
+							'description' => __( 'Image alternative text.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+					],
+				],
+			],
+			'variation' => [
+				'description' => __( 'Chosen attributes (for variations).', 'woo-gutenberg-products-block' ),
+				'type'        => 'array',
+				'context'     => [ 'view', 'edit' ],
+				'items'       => [
+					'type'       => 'object',
+					'properties' => [
+						'attribute' => [
+							'description' => __( 'Variation attribute name.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+						],
+						'value'     => [
+							'description' => __( 'Variation attribute value.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+						],
+					],
+				],
+			],
+			'totals'    => [
+				'description' => __( 'Item total amounts provided using the smallest unit of the currency.', 'woo-gutenberg-products-block' ),
+				'type'        => 'object',
+				'context'     => [ 'view', 'edit' ],
+				'readonly'    => true,
+				'properties'  => array_merge(
+					$this->get_store_currency_properties(),
+					[
+						'line_subtotal'     => [
+							'description' => __( 'Line price subtotal (excluding coupons and discounts).', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'line_subtotal_tax' => [
+							'description' => __( 'Line price subtotal tax.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'line_total'        => [
+							'description' => __( 'Line price total (including coupons and discounts).', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'line_total_tax'    => [
+							'description' => __( 'Line price total tax.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+					]
+				),
+			],
+		];
+	}
+
+	/**
+	 * Check given ID exists,
+	 *
+	 * @param integer $product_id Product ID.
+	 * @return bool
+	 */
+	public function product_id_exists( $product_id ) {
+		$post = get_post( (int) $product_id );
+		return $post && in_array( $post->post_type, [ 'product', 'product_variation' ], true );
+	}
+
+	/**
+	 * Convert a WooCommerce cart item to an object suitable for the response.
+	 *
+	 * @todo Variation is stored to meta - how can we gather for response?
+	 *
+	 * @param \WC_Order_Item_Product $line_item Order line item array.
+	 * @return array
+	 */
+	public function get_item_response( $line_item ) {
```

Comment:
Maybe typehint here for `\WC_Order_Item_Product` to make it clear the expectation?